src/Main.java

```diff
@@ -2,27 +2,52 @@
 import java.util.*;
 import java.util.stream.IntStream;
 
+class User {
```

Comment:
Make user as separate class ( I mean create .java class in inteleji, such as Main.java)